flowforge-container/Dockerfile

```diff
@@ -4,7 +4,8 @@ ARG REGISTRY
 ARG TAG
 RUN if [[ ! -z "$REGISTRY" ]] ; then npm config set @flowforge:registry "$REGISTRY"; fi
 
-RUN apk add --no-cache --virtual build-base g++ make py3-pip sqlite-dev python2
+RUN apk add --no-cache --virtual build-base g++ make py3-pip sqlite-dev python3
+RUN ln -s `which python3` /usr/bin/python
```

Comment:
Instead of linking maybe consider an alias...
`alias python=python3` or `alias python='/usr/bin/python3'` 

Or,  install `python-is-python3` instead? 

Just a suggestion - you know how the container is build so please feel free to ignore suggestions.

Reply:
python-is-python3 is ubuntu only, the link makes sure things work if there is no shell for alias so I'm going to stick with this for now.

Reply:
Ben has confirmed the symlink will not clash so good to go :)